ts/Dashboards/Layout/Layout.ts

```diff
@@ -418,6 +418,28 @@ class Layout extends GUIElement {
             }
         };
     }
+
+    /**
+     * Get the layout's options.
+     * @returns
+     * The JSON of layout's options.
+     *
+     * @internal
+     *
+     */
+    public getOptions(): DeepPartial<Layout.Options> {
+        const layout = this,
+            rows = [];
+
+        // Get rows JSON.
+        for (let i = 0, iEnd = layout.rows.length; i < iEnd; ++i) {
+            rows.push(layout.rows[i].getOptions());
+        }
+
+        return {
+            rows
+        };
```

Comment:
maybe let's serialise some options as well, like `id`, `rowClassName`, `cellClassName`, `layoutClassName` (they are in the API and in the interface)?